src/ludamus/adapters/db/django/migrations/0127_alter_party_invite_token.py

```
@@ -0,0 +1,26 @@
+    dependencies = [("db_main", "0126_user_manager")]
+
+    operations = [
+        migrations.AlterField(
+            model_name="party",
+            name="invite_token",
+            field=models.CharField(
+                db_index=True, default=secrets.token_urlsafe, max_length=64
+            ),
+        ),
+        migrations.RunPython(backfill_invite_tokens, migrations.RunPython.noop),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)